cmd/keycmd/list.go

```diff
@@ -119,18 +119,13 @@ func getClients(networks []models.Network, cchain bool) (
 	map[models.Network]ethclient.Client,
 	error,
 ) {
-	apiEndpoints := map[models.Network]string{
-		models.Fuji:    constants.FujiAPIEndpoint,
-		models.Mainnet: constants.MainnetAPIEndpoint,
-		models.Local:   constants.LocalAPIEndpoint,
-	}
 	var err error
 	pClients := map[models.Network]platformvm.Client{}
 	cClients := map[models.Network]ethclient.Client{}
 	for _, network := range networks {
-		pClients[network] = platformvm.NewClient(apiEndpoints[network])
+		pClients[network] = platformvm.NewClient(network.Endpoint)
 		if cchain {
-			cClients[network], err = ethclient.Dial(fmt.Sprintf("%s/ext/bc/%s/rpc", apiEndpoints[network], "C"))
+			cClients[network], err = ethclient.Dial(fmt.Sprintf("%s/ext/bc/%s/rpc", network.Endpoint, "C"))
```

Comment:
nit: provide func to return `fmt.Sprintf("%s/ext/bc/%s/rpc", network.Endpoint, "C")` as part of the model Network